src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -630,46 +635,23 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
       return { icon: '􀕹', label: review, disabled: true };
     }
 
-    const inputAsset = internalSelectedInputAsset.value;
-    const sellAmount = (() => {
```

Comment:
previously: this `sellAmount` was returning two different orders of magnitude in the response which was incorrect.
e.g, when there's a quote error, it would return the input amount (like "1.234") but if there's a quote, it would return the raw sell amount (like "12340")